code/game/objects/items/devices/radio/radio.dm

```diff
@@ -273,13 +273,14 @@ var/global/list/default_medbay_channels = list(
 	if (!connection)
 		return
 
-	var/mob/living/silicon/ai/A = new /mob/living/silicon/ai(src, null, null, 1)
-	A.SetName(from)
-	Broadcast_Message(connection, A,
-						0, "*garbled automated announcement*", src,
-						message, from, "Automated Announcement", from, "synthesized voice",
-						4, 0, list(0), connection.frequency, "states")
-	qdel(A)
+	var/mob/living/announcer/M = new(src)
+	M.PrepareBroadcast(from)
+	Broadcast_Message(connection, M,
+						FALSE, "*garbled automated announcement*", src,
+						message, from, "Automated Announcement", from, M.voice_name,
+						4, 0, list(0), connection.frequency, "states", M.default_language)
+	// M.ResetAfterBroadcast()
```

Comment:
Commented out code?

Reply:
Yeah, I specifically put that in because that's what you're supposed to run whenever you change the Announcer mob with its `PrepareBroadcast` proc - in case someone wanted to make this mob persistent so they don't forget. But it's not needed now since it's just local and gets deleted, so - commented out.